pkg/alertmanager/alertstore/local/store.go

```diff
@@ -148,9 +149,40 @@ func (f *Store) reloadConfigs() (map[string]alertspb.AlertConfigDesc, error) {
 		// The file name must correspond to the user tenant ID
 		user := strings.TrimSuffix(info.Name(), ext)
 
+		// Load template files
+		userTemplateDir := filepath.Join(f.cfg.Path, user, templatesDir)
+		var templates []*alertspb.TemplateDesc
+
+		if _, e := os.Stat(userTemplateDir); e == nil {
```

Comment:
We should probably check for `e` is not `os.ErrNotExist` then return error. I think `os.Stat(userTemplateDir)`may return other errors like "no permission" etc. which we should propagate up.  There is also [`os.IsExist()` helper function](https://pkg.go.dev/os#IsExist).